ONSdigital/rm-collection-exercise-service
# Remove bi/transition straight to live if go live in past for execution

# Motivation and Context

Solves the issue of collection exercises with go live dates in the past transition to live instead of ready for live.

# What has changed

Now when a collection exercise in a validated state attempts to transition to ready for live if the go live date has past then the collection exercise is transitioned to live state.



# How to test?

Run make set up upload a collection instrument and sample for a survey in ROps with pre-populated dates (make sure go live date in past) then set ready for live should go directly to live.



# Links

[Trello] https://trello.com/c/PLkePSlW
